person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1..10_000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    true if is_of_age(@age) == true || parent_permission == true
+  end
```

Comment:
- [ ] This method can_use_services? should be out of your private methods section. Means it should have been created before line 12 👍 to make it accessible by the user 😊
- [ ] In line 19, you are using **is_of_age** method which doesn't exist. You have made method which is **of_age?**. Kindly use that 👍